src/superqt/utils/_animation.py

```diff
@@ -0,0 +1,76 @@
+"""Different animators to be used for animating components"""
+# from math import sin, cos, radians
+
+from ..qtcompat.QtCore import QEasingCurve, QPropertyAnimation, QVariantAnimation
+from ..qtcompat.QtGui import QIcon, QPixmap, QTransform
+from ..qtcompat.QtWidgets import QAbstractButton, QWidget
+
+
+# =================================================================================================
+def create_hide_show_animation(
```

Comment:
while I like the idea of these utility functions, I'm not sure they're general enough to live on their own.  That is, this is basically just using the `QPropertyAnimation` api directly, not _adding_ something that Qt doesn't already provide itself.  

Since this function is only used in a single place here, I think for now I'd prefer moving this function into the module where it is used, and prefixing it with an underscore to keep it private (so we're not implying that this is part of the general superqt API just yet)

Reply:
Agree. I debated this myself. I was not sure and forgot to add it as a note in the PR. migrated.